### PR TITLE
Correcting x-ray scattering factors

### PR DESCRIPTION
**Pull request by thamnos**
The file scatterfactors-xray-table did not contain the factors for x-ray scattering but for *electron* scattering (as was correctly stated in the file header).
In this corrected version, many x-ray scattering factors have been copied by hand from the International Tables
[20170711144633429.pdf](https://github.com/camm/sassena/files/1138874/20170711144633429.pdf)
As the values are identical to the scatterfactors-xray-table-SERENA, that file was removed.

As in the previous version, these values are valid for values of s up to 6.0A^{-1} and therefore Q=2*pi*s up to approx 38A^{-1}.

We also transcribed the values for ions in the hope that sassena does not pick them up at the moment by mistake (e.g. through a too general regex).

### CODE REVIEW

File: tests/waterdyn4/database/definitions/scatterfactors-xray-table.xml [hunk 1]
```diff
@@ -2,585 +2,925 @@
...
+			<param>6.70003	</param>
+			<param>0.205855	</param>
+			<param>6.06791	</param>
+			<param>5.47913	</param>
+			<param>0.859041	</param>
+			<param>11.6030	</param>
+			<param>0.393974	</param>
+		</element> 
+		<element>
+			<name>As	 <!-- wrong data  -->		</name>
```
Comment (jmborr) [inline tests/waterdyn4/database/definitions/scatterfactors-xray-table.xml]: What is the meaning of `wrong data` here? In the PDF you linked to there's data for `As` (page 573) so why not include that data here?

**Comment by thamnos** [comment]
We've deleted the old (wrong) data for the minute. I will see if I can find a volunteer to type up the rest... That might take some time though.

**Comment by stuartcampbell** [comment]
Would it make sense to use something like https://github.com/scikit-beam/XrayDB instead ?

**Comment by jmborr** [comment]
@stuartcampbell to be honest I can't comment on this because I know very little of X-rays. Do you think users of BNL would be interested in using SASSENA to compare simulations to X-ray data?

**Pull request merged**